lib/ansible/modules/cloud/amazon/aws_organizations.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Kamil Potrec <kamilpotrec@googlemail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+module: aws_organizations
+short_description: Manages AWS Organization
+description:
+  - Creates and deletes AWS OrganizationalUnits
+version_added: "2.10"
+author:
+  - Kamil Potrec (@p0tr3c)
+options:
+  name:
+    description:
+      - Name of the organizational unit
+    required: true
+    type: str
+  state:
+    description:
+      - State of the organizational unit
+    default: present
+    type: str
+    choices:
+      - present
+      - absent
+      - describe
+      - list
```

Comment:
While you may have encountered existing modules that combine functionality like this, it is not allowed in new modules (see https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_best_practices.html#scoping-your-module-s). list/describe should be split into a separate _info module.

Reply:
Thanks for quick feedback. I will remove the info functionality, thank you for pointing it out.
I've seen a pull request which proposes _info module (#55837), so I won't create a new one at the time.